README.md

```diff
@@ -30,7 +30,7 @@ How to install and run Brackets
 
 Installers for the latest stable build for Mac, Windows and Linux (Debian/Ubuntu) can be [downloaded here](http://brackets.io/).
 
-The Linux version has most of the features of the Mac and Windows versions, but
+The Linux version has more features than the Mac and Windows versions, but
```

Comment:
This change changes the meaning of the sentence: it used to be that only some of the features of the Mac/Windows versions are present in the Linux version. Now, more features are present in the Linux version than the Mac/Windows version.

Is this the case? I would doubt that - I would suspect that Mac and Windows have more features than the Linux version.